components/swatchgroup/stories/swatchgroup.stories.js

```diff
@@ -68,15 +115,94 @@ export default {
 	},
 };
 
-export const Default = SwatchGroup.bind({});
+/**
+ * The default swatch group has regular density.
+ */
+export const Default = SwatchgroupGroup.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+export const Compact = Template.bind({});
+Compact.args = {
+	density: "compact",
+};
+Compact.tags = ["!dev"];
+Compact.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const Spacious = Template.bind({});
+Spacious.args = {
+	density: "spacious",
+};
+Spacious.tags = ["!dev"];
+Spacious.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Only use rounded swatches if there is a single row.
+ */
+export const Rounding = RoundingTemplate.bind({});
+Rounding.args = {};
+Rounding.tags = ["!dev"];
+Rounding.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Use any size swatches as necessary. The medium size is the default option. This only affects the size of each individual swatch, not the spacing between them.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withBorder: false,
+	withHeading: false, 
+	...args,
+}, context);
+Sizing.args = Default.args;
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * In this example, the lower contrast swatches utilize the `.spectrum-Swatch--lightBorder` class.
```

Comment:
I was thinking the same while reviewing. I think it'd be a good idea to use a different set of colors that are all above that 3:1 ratio for all of the swatch groups other than the light border example. I saw that the yellow for example is only 1.1:1.

Reply:
I am going to have to ask for a pairing session- I'm a little lost on how to handle all of the light border stuff, and then making sure I pick colors that are contrast-y enough for both light & dark modes. I'll slack you both.

Reply:
Here's the updates we talked about in our group pairing. 

- new colors for most of the stories with colors that meet contrast in both light & dark mode
- refactored the rounding template to just use the same set of colors for each variation
- new colors for the lightBorder story and test case. I could only find colors that meet contrast for either light OR dark mode, so I went with light mode and mentioned that in the description.
- Updated and (hopefully) further clarified the light border documentation. Still open to help with that if you think it can be improved.

https://github.com/adobe/spectrum-css/pull/2925/commits/cacd28169991af8648e1309a0e6bca797a3a186d